system-tests/lib/crecli/commands.go

```diff
@@ -16,6 +19,77 @@ type CompilationResult struct {
 	SecretsURL  string
 }
 
+func SetFeedAdmin(creCLICommandPath string, chainID int, adminAddress common.Address, settingsFile *os.File) error {
```

Comment:
We have `ConfigureDataFeedsCache` that performs `SetFeedAdminConfig` and `SetFeedDecimalConfig` too. They look the same, so why do we need both? @Tofel @karen-stepanyan

Reply:
Let's use the CLI for now since we don't know the release cycles of the CLI; more testing fidelity is better.